tomviz/PresetModel.cxx

```diff
@@ -0,0 +1,78 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#include "PresetModel.h"
+
+#include <pqPresetToPixmap.h>
+
+#include <QPair>
+#include <QPixmap>
+#include <QScopedPointer>
+#include <QSize>
+
+namespace tomviz {
+
+// ctor
+PresetModel::PresetModel(QObject *parent)
+  : QAbstractTableModel(parent)
+{
+  //  vtkNew<vtkSMTransferFunctionPresets> Presets;
+  auto begin = Presets->GetNumberOfPresets();
+  QString file = "../tomviz/tomviz/resources/matplotlib_cmaps.json";
```

Comment:
You might be able to either copy-paste or re-use [this function](https://github.com/OpenChemistry/tomviz/blob/6d07b4cb05b9a18e794fb27d0dc75f5e80bc7e7f/tomviz/Behaviors.cxx#L113) to make the json file more easily obtainable.

Reply:
As Patrick pointed out, this needs to be relative to the application directory, and doesn't work for mo locally. It would be good to adapt the code he pointed out, this wouldn't work in an installed application either.